src/main/java/edu/hawaii/its/api/service/UpdateMemberService.java

```diff
@@ -114,6 +113,15 @@ public GroupingRemoveResult removeOwnership(String currentUser, String groupingP
         return removeOwner(currentUser, groupingPath, validIdentifier);
     }
 
+    public GroupingRemoveResults removeGroupPathOwnerships(String currentUser, String groupingPath,
+            List<String> groupPathOwner) {
+        log.info(String.format("removeOwnerships; currentUser: %s; groupingPath: %s; uhIdentifiers: %s;",
```

Comment:
I believe this string should start with `removeGroupPathOwnerships`, not `removeOwnerships`.